### PR TITLE
Support ruby 2.4

### PR DESCRIPTION
**Pull request by namusyaka**
[Fixnum is deprecated now](https://bugs.ruby-lang.org/issues/12005), so I guess we should drop Fixnum and use Integer instead.
@dblock @dm1try Thoughts?

### CODE REVIEW

**Comment by dblock** [comment]
Support for Ruby 2.4 will have to be called out in README since it doesn't work out of the box.

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +7,16 @@ matrix:
     - rvm: 2.3.3
```
Comment (LeFnord) [inline .travis.yml]: please can you change it to `2.4.0`

**Comment by LeFnord** [comment]
👍 … thanks

File: .travis.yml (hunk 1 above)
Comment (namusyaka) [inline .travis.yml]: Right. Thanks.

Comment (namusyaka) [inline .travis.yml]: Fixed.

File: README.md [hunk 2]
```diff
@@ -799,6 +800,28 @@ The following are all valid types, supported out of the box by Grape:
...
+  params[:integers][:int].class
+end
+
+...
+
+get '/int' integers: { int: '45' }
+  #=> Integer in ruby 2.4
+  #=> Fixnum in earlier ruby versions
+```
+
```
Comment (namusyaka) [inline README.md]: Added Integer/Fixnum and Corcions section. Could you review this ?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,13 @@
-#### 0.19.1 (Next)
+### 0.19.1 (Next)
+
+#### Features
 
 * [#1536](https://github.com/ruby-grape/grape/pull/1536): Updates `invalid_versioner_option` translation - [@Lavode](https://github.com/Lavode).
+* [#1543](https://github.com/ruby-grape/grape/pull/1543): Support ruby 2.4 - [@LeFnord](https://github.com/LeFnord), [@namusyaka](https://github.com/namusyaka).
```
Comment (namusyaka) [inline CHANGELOG.md]: Added @LeFnord .

**Comment by LeFnord** [comment]
… it seems, it could be mörged …

File: spec/support/integer_helpers.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+module Spec
+  module Support
+    module Helpers
+      INTEGER_CLASS_NAME = 0.to_i.class.to_s.freeze
```
Comment (dblock) [inline spec/support/integer_helpers.rb]: How clever!

**Comment by dblock** [comment]
Merged.

**Pull request merged**